models/Customer.js

```diff
@@ -2,15 +2,15 @@ var keystone = require('keystone');
 var Types = keystone.Field.Types;
 
 /**
- * Customer Model
- * ==========
+ Error occurred when you try to add customer info in admin page.
```

Comment:
what does this mean?
why do you want to add customer info in admin page?
admin page means keystone admin page, right?

Reply:
When I tried to add customer info in keystone admin page for test, it didn't work for some reason.

Reply:
Right, when user contact company and ask something's wrong, then admin has to look into their informations. yes.. so customer info should be available on admin page. I see.

I'll also see what I can do

Reply:
Try this. I removed "unique: true" from uid field.
error occurred because you tried to add customer which uid is null, when there's already customer which uid is null. 
since we we added unique option for uid, there cannot be more than two customers that uid is null

-----------------------------------------------------------------------
Customer.add({
	uid: { type: Types.Text },
	name: { type: Types.Name, required: true },
	email: { type: Types.Email, initial: true, required: true, unique: true },
	phoneNumber: { type: Types.Text },
	address: {
		address1: { type: Types.Text },
		address2: { type: Types.Text },
		city: { type: Types.Text },
		state: { type: Types.Text },
		zipcode: { type: Types.Number },
	}
});